daffodil-cli/src/main/scala/org/apache/daffodil/debugger/CLIDebuggerRunner.scala

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2012-2015 Tresys Technology, LLC. All rights reserved.
```

Comment:
We'll typically only change package name once the SGA is on file and we can remove copyright.

Reply:
Ok. So maybe we should hold off on the two commits that change the directory layout and package names/XML namespaces for now.